storage/client/file/src/main/java/com/azure/storage/file/FileStorageClient.java

```diff
@@ -17,53 +17,53 @@
 import java.util.Map;
 
 /**
- * This class provides a fileServiceAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
- * Operations allowed by the fileServiceAsyncClient are creating, listing, and deleting shares and retrieving and updating properties
+ * This class provides a fileStorageAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
+ * Operations allowed by the fileStorageAsyncClient are creating, listing, and deleting shares and retrieving and updating properties
  * of the account.
  *
  * <p><strong>Instantiating a Synchronous File Service Client</strong></p>
  *
- * {@codesnippet com.azure.storage.file.fileServiceClient.instantiation}
+ * {@codesnippet com.azure.storage.file.fileStorageClient.instantiation}
  *
- * <p>View {@link FileServiceClientBuilder this} for additional ways to construct the fileServiceAsyncClient.</p>
+ * <p>View {@link FileStorageClientBuilder this} for additional ways to construct the fileStorageAsyncClient.</p>
  *
- * @see FileServiceClientBuilder
- * @see FileServiceAsyncClient
+ * @see FileStorageClientBuilder
+ * @see FileStorageAsyncClient
  * @see SharedKeyCredential
  * @see SASTokenCredential
  */
-public final class FileServiceClient {
-    private final FileServiceAsyncClient fileServiceAsyncClient;
+public final class FileStorageClient {
+    private final FileStorageAsyncClient fileStorageAsyncClient;
 
     /**
-     * Creates a FileServiceClient that wraps a FileServiceAsyncClient and blocks requests.
+     * Creates a FileStorageClient that wraps a FileStorageAsyncClient and blocks requests.
      *
-     * @param client FileServiceAsyncClient that is used to send requests
+     * @param client FileStorageAsyncClient that is used to send requests
      */
-    FileServiceClient(FileServiceAsyncClient client) {
-        this.fileServiceAsyncClient = client;
+    FileStorageClient(FileStorageAsyncClient client) {
+        this.fileStorageAsyncClient = client;
     }
 
     /**
      * Get the url of the storage file service client.
      * @return the url of the Storage File service.
      * @throws RuntimeException If the directory is using a malformed URL.
      */
-    public URL getFileServiceUrl() {
-        return fileServiceAsyncClient.getFileServiceUrl();
+    public URL getFileStorageUrl() {
+        return fileStorageAsyncClient.getFileStorageUrl();
     }
 
     /**
      * Constructs a ShareClient that interacts with the specified share.
      *
-     * <p>If the share doesn't exist in the storage account {@link ShareClient#create() create} in the fileServiceAsyncClient will
+     * <p>If the share doesn't exist in the storage account {@link ShareClient#create() create} in the fileStorageAsyncClient will
```

Comment:
This should state that the create method in the returned ShareClient needs to be called before interacting with the share can happen/will work.

Reply:
Good catch.